pkg/controller/machine.go

```diff
@@ -361,7 +361,8 @@ func (c *controller) machineCreate(machine *v1alpha1.Machine, driver driver.Driv
 		// Get the latest version of the machine so that we can avoid conflicts
 		machine, err := c.controlMachineClient.Machines(machine.Namespace).Get(machine.Name, metav1.GetOptions{})
 		if err != nil {
-			return err
+			glog.Warningf("Machine GET failed. Retrying, error: %s", err)
```

Comment:
Could the name of the machine be useful in the logs?

Reply:
Yes, makes sense. Made suggested changes. Refer - https://github.com/gardener/machine-controller-manager/pull/330/commits/c3b4dec3a8cd10096d8464fc3550999a630571d9.